<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="startGame">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>MOVE! (P1)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitGame">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>320</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>QUIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonFire">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>120</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FIRE</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="playerTurn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Player 1's Turn</string>
   </property>
  </widget>
  <widget class="QLabel" name="moveP">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="chooseMove">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose Move!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonWater">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>120</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>WATER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonGrass">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>120</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GRASS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonHeal">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>120</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>HEAL</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBlock">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>120</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BLOCK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSwap">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>120</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SWAP</string>
   </property>
  </widget>
  <widget class="QLabel" name="player1health">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>192</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="player2health">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>192</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BusyCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="playerStats">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="chooseMove_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ELEMENT OF SURPRISE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="shoot">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>SHOOT</string>
   </property>
  </widget>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playMusic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Restart Music</string>
   </property>
  </widget>
  <zorder>bg</zorder>
  <zorder>startGame</zorder>
  <zorder>quitGame</zorder>
  <zorder>imgLabel</zorder>
  <zorder>buttonFire</zorder>
  <zorder>playerTurn</zorder>
  <zorder>moveP</zorder>
  <zorder>chooseMove</zorder>
  <zorder>buttonWater</zorder>
  <zorder>buttonGrass</zorder>
  <zorder>buttonHeal</zorder>
  <zorder>buttonBlock</zorder>
  <zorder>buttonSwap</zorder>
  <zorder>player1health</zorder>
  <zorder>player2health</zorder>
  <zorder>playerStats</zorder>
  <zorder>chooseMove_2</zorder>
  <zorder>shoot</zorder>
  <zorder>playMusic</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
